examples/light-switch-app/silabs/src/LightSwitchMgr.cpp

```diff
@@ -125,6 +232,26 @@ void LightSwitchMgr::TriggerLightSwitchAction(LightSwitchAction action, bool isG
     DeviceLayer::PlatformMgr().ScheduleWork(SwitchWorkerFunction, reinterpret_cast<intptr_t>(data));
 }
 
+void LightSwitchMgr::TriggerLevelControlAction(uint8_t stepMode, bool isGroupCommand)
```

Comment:
The signature should be the type that is used we don't need the double conversion.

Reply:
changed the signature